### DISCUSSION TITLE
How to work collaboratively on a project that uses space dev?

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/space-docs#105] [Q&A] [answered]
I use `space dev` on my project. I would like to work on this with my team, but how can I make it so that they can use `space dev`, because (as I understand it) it directly links to the real, production Deta Bases and Deta Drives. I am the only one that can access this.

If I am wrong please tell me.

Also, is it able to make sure that it doesn't affect the production Deta Bases?

**Comment by s4m-mo** [comment]
i.e. How can I create and manage a developer branch of my Deta Base, separately from my production Deta Base?

**Reply by SlumberDemon** [reply, marked as answer]
> because (as I understand it) it directly links to the real, production Deta Bases and Deta Drives

No, when you develop your space app and test it locally with `space dev` you connect to your dev instance which has its own Base/Drive.

> Also, is it able to make sure that it doesn't affect the production Deta Bases?

Every space app has a clean Base and Drive when installing it (if published in discovery)

*I would recommend taking a look at this: https://deta.space/docs/en/learn/what-is-pc*
